client/src/components/landing/landing.jsx

```diff
@@ -23,7 +23,7 @@ function Landing() {
 
   return (
     <>
-      { (token === '') ? <Login/> : <Player token={token} refreshToken={refreshToken}/>}
+      { (token === '') ? <Login/> : <Player token={token} refreshToken={refreshToken} setUsername={setUsername}/>}
```

Comment:
I think that we should have a `<Room>` component to encompass the `<Player>` and all other components of a room (e.g. Queue, SearchBar, Chat). Thus we'd only need to pass down setUsername() to the `<Room>` component, and then that room component can pass down stuff to each respective child.